src/python/pants/backend/jvm/tasks/coverage/manager.py

```diff
@@ -57,21 +61,24 @@ class CodeCoverage(Subsystem):
 
   @classmethod
   def subsystem_dependencies(cls):
-    return super().subsystem_dependencies() + (Cobertura.Factory, Jacoco.Factory)
+    return super().subsystem_dependencies() + (Cobertura.Factory, Jacoco.Factory, Scoverage.Factory)
 
   # TODO(jtrobec): move these to subsystem scope after deprecating
   @staticmethod
   def register_junit_options(register, register_jvm_tool):
     register('--coverage', type=bool, fingerprint=True, help='Collect code coverage data.')
     register('--coverage-processor', advanced=True, fingerprint=True,
-             choices=['cobertura', 'jacoco'], default=None,
+             choices=['cobertura', 'jacoco', 'scoverage'], default=None,
              help="Which coverage processor to use if --coverage is enabled. If this option is "
-                  "unset but coverage is enabled implicitly or explicitly, defaults to 'cobertura'."
-                  "If this option is explicitly set, implies --coverage.")
+                  "unset but coverage is enabled implicitly or explicitly, defaults to 'cobertura'. "
+                  "If this option is explicitly set, implies --coverage. If this option is set to "
+                  "scoverage, then first scoverage MUST be enabled by passing option "
+                  "--scoverage-enable-scoverage.")
     # We need to fingerprint this even though it nominally UI-only affecting option since the
     # presence of this option alone can implicitly flag on `--coverage`.
     register('--coverage-open', type=bool, fingerprint=True,
-             help='Open the generated HTML coverage report in a browser. Implies --coverage.')
+             help='Open the generated HTML coverage report in a browser. Implies --coverage '
+                  'with cobertura as the engine.')
```

Comment:
Is there any reason that this can't be applied for scoverage as well? But also, it's supported by both `jacoco` and `cobertura`.

Reply:
No I just modified the comment here because if a user passes JUST `--test-junit-coverage-open`, then cobertura is used by default. I can change it back if it doesn't help.

Reply:
I guess it didn't make much sense. Changed it back.